pkg/util/kubernetes/kubelet/container_test.go

```diff
@@ -159,3 +160,20 @@ func (suite *ContainersTestSuite) TestParseContainerReadiness() {
 func TestContainersTestSuite(t *testing.T) {
 	suite.Run(t, new(ContainersTestSuite))
 }
+
+func TestKubeContainerIDToEntityID(t *testing.T) {
+	for in, out := range map[string]string{
+		"container_id://deadbeef": "container_id://deadbeef",
+		"containerd://deadbeef":   "container_id://deadbeef",
+		"cri-o://deadbeef":        "container_id://deadbeef",
+		"cri-o://d":               "container_id://d",
+		"container_id://":         "",
+		"deadbeef":                "",
+		"/deadbeef":               "",
```

Comment:
We should add a test `runtime://foo/bar` to make it explicit that we expect `container_id://foo/bar`